From 32adc0a77c23521b46ccc1821e5eb6359c43c008 Mon Sep 17 00:00:00 2001
From: Kevin Brace <kevinbrace@gmx.com>
Date: Fri, 8 Jul 2016 00:23:31 -0500
Subject: [PATCH 10/21] Version bumped to 0.5.102

I need to thank Eric Kudzin for bringing the equipment to help me
fix this bug.

Signed-off-by: Kevin Brace <kevinbrace@gmx.com>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index c359321..4bd91d1 100644
--- a/configure.ac
+++ b/configure.ac
@@ -23,7 +23,7 @@
 # Initialize Autoconf
 AC_PREREQ(2.57)
 AC_INIT([xf86-video-openchrome],
-        [0.5.101],
+        [0.5.102],
         [https://bugs.freedesktop.org/enter_bug.cgi?product=xorg&component=Driver/openchrome],
         [xf86-video-openchrome])
 
-- 
1.9.1

